### ISSUE TITLE
tomoscan_stream_2bm.py does not actually use pvaccess?

### ISSUE THREAD

**Issue by MarkRivers** [tomography/tomoscan#98]
tomoscan_stream_2bm.py imports pvaccess.  But it does not appear to me that it actually uses pvaccess from Python, all it does is configure the NDPluginPva plugin in areaDetector.

If this is true then the "import pvaccess" line should be removed.  It caused tomoscan to fail to run on Windows, because it loads a version of ca.dll that is incompatible with the default version used by PyEpics.

This is discussed in this EPICS tech-talk thread. https://epics.anl.gov/tech-talk/2021/msg00147.php

**Comment by decarlof** [comment]
That is a mistake and should be removed. We  [moved pva setup to tomoscan.py](https://github.com/tomography/tomoscan/commit/3675c4bf99b6a05dbd4357bda8a865113f7dab0f) from tomoscan_stream_2bm.py and in the process I forgot to remove the `import pvaccess`.

We still need `import pvaccess` in the base class. Is this OK?

**Comment by MarkRivers** [comment]
Why was that moved to tomoscan.py?  It seems like it belongs in the class that uses it?

**Comment by decarlof** [comment]
that is correct, is [work in progress](https://github.com/tomography/tomoscan/pull/97). I should be able to push it in the next couple of days. We are creating a tomoscan_stream.py and all generic function associated to tomoscan_stream will go there including the:

```
import pvaccess

...
        # Setting the pva servers to broadcast dark and flat fields
        if 'PvaStream' in self.pv_prefixes:
            prefix = self.pv_prefixes['PvaStream']
            self.pva_stream_dark = pvaccess.PvObject({'value': [pvaccess.pvaccess.ScalarType.FLOAT], 
                'sizex': pvaccess.pvaccess.ScalarType.INT, 
                'sizey': pvaccess.pvaccess.ScalarType.INT})
            self.pva_server_dark = pvaccess.PvaServer(prefix + 'dark', self.pva_stream_dark)

            self.pva_stream_flat = pvaccess.PvObject({'value': [pvaccess.pvaccess.ScalarType.FLOAT], 
                'sizex': pvaccess.pvaccess.ScalarType.INT, 
                'sizey': pvaccess.pvaccess.ScalarType.INT})
            self.pva_server_flat = pvaccess.PvaServer(prefix + 'flat', self.pva_stream_flat)


```

**Comment by MarkRivers** [comment]
Sounds good.

**Comment by decarlof** [comment]
this has been addressed by #97
